ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -718,6 +719,22 @@ public KsqlConfig overrideBreakingConfigsWithOriginalValues(final Map<String, St
     return new KsqlConfig(ConfigGeneration.LEGACY, mergedProperties, mergedStreamConfigProps);
   }
 
+  public Map<String, String> getStringAsMap(final String key) {
+    final String value = getString(key).trim();
+    try {
+      return value.equals("")
```

Comment:
Check for the NULL value as well. I think if you set a config like `config=` without a value, then the value returned will be null. I recalled seen this null value in some tests with configs.

Reply:
Hm, I tried setting `ksql.metrics.tags.custom=` just now and got the empty string. Are you able to reproduce getting a null value?

Reply:
Perhaps it was a different configuration that has NULL as default.